### PR TITLE
Hot fix for temporalValues

### PR DESCRIPTION
**Pull request by hiteshNEC**

### CODE REVIEW

File: History/HistoryManager/src/main/java/eu/neclab/ngsildbroker/historymanager/repository/HistoryDAO.java [hunk 1]
```diff
@@ -129,9 +129,12 @@ protected String translateNgsildQueryToSql(QueryParams qp) throws ResponseExcept
 			//}
 			sqlQuery += "}')::jsonb";
 		}
-		sqlQuery += "  as tedata, " + "jsonb_object_agg(attributeid, attributedata";
+		sqlQuery += "  as tedata, " + "jsonb_object_agg(attributeid,";
 		if (qp.getTemporalValues()) {
-			sqlQuery += "->0->'" + NGSIConstants.NGSI_LD_HAS_VALUE + "'->0->'" + NGSIConstants.JSON_LD_VALUE + "'";
+			sqlQuery += "(select json_agg(jsonb_build_array(t -> '"+NGSIConstants.NGSI_LD_HAS_VALUE+"'->0->'" + NGSIConstants.JSON_LD_VALUE + "'"+",t->'https://uri.etsi.org/ngsi-ld/observedAt'->0->'@value')) from jsonb_array_elements(attributedata) as x(t))";
```
Comment (amitrnec) [inline History/HistoryManager/src/main/java/eu/neclab/ngsildbroker/historymanager/repository/HistoryDAO.java:134]: [Minor]Please "https://uri.etsi.org/ngsi-ld/observedAt" get from NGSIConstants same as other .

Comment (amitrnec) [inline History/HistoryManager/src/main/java/eu/neclab/ngsildbroker/historymanager/repository/HistoryDAO.java:134]: [Minor]Get "@value" from NGSIConstants also.

**Review by ScorpioBroker — CHANGES_REQUESTED** [review]
Hi,
yes please change the strings in general to the matching constants.
About the observedAt. This should be the time property from the query parameter here. observedAt is just the default (which qp should already handle if i recall right).

**Comment by hiteshNEC** [comment]
> Hi,
> yes please change the strings in general to the matching constants.
> About the observedAt. This should be the time property from the query parameter here. observedAt is just the default (which qp should already handle if i recall right).

**Review by amitrnec — APPROVED** [review]

**Comment by hiteshNEC** [comment]
> yes please change the strings in general to the matching constants.
Done
> About the observedAt. This should be the time property from the query parameter here. observedAt is just the default (which qp should already handle if i recall right).
According to the spec section C.5.6, we have to return value with their observedAt.

**Comment by ScorpioBroker** [comment]
@hiteshNEC C5.6 shows an example where no temporalproperty is provided and hence defaults to observedAt. Please check 4.11 for more details on that.

**Review by ScorpioBroker — APPROVED** [review]
lgtm

**Pull request merged**